README.md

```diff
@@ -23,15 +26,41 @@ The following apps are still under development, not ready for production deploym
 npm install
 ```
 
-This installs global package dependencies and also bootstraps lerna packages.
+This installs global package dependencies and also bootstraps the entire monorepo through [`lerna`](https://github.com/lerna/lerna).
+
+> **Note**: the monorepo is set up in such a way that you **must** install it through a `lerna bootstrap` (done automatically after an `npm install`).
+>
+> If you're only interested in the contract dependencies, and not the frontends, you can use `INSTALL_FRONTEND=false npm install` instead.
+>
+> If you're only interested in bootstrapping one package, you can use `npx lerna bootstrap --scope @aragon/<package> --include-filtered-dependencies`
+
+Running tests on all apps can be done by running `npm run test` at the root directory (note that running all of the tests can take a significant amount of time!).
+
+Running tests of an individual app can be done by running `npm run test` inside an individual app's directory, or through the selective `npm run test:<app>` scripts.
+
+By default, tests are run in a in-memory instance of testrpc.
+
+### App frontends
+
+Developing the frontends locally can be a bit tricky, due to the frontends' reliance on a properly configured Aragon client.
+
+For now, you can follow [@bpierre's gist](https://gist.github.com/bpierre/22572415b90577aa17e5953fa68980eb#developing-the-apps) to start an app's local frontend server and connect it to a locally running Aragon client.
```

Comment:
Should we start to move this in every frontend-related repository, with a name like `CONTRIBUTING_FRONTEND.md` or `QUICKSTART_FRONTEND.md`, even if it’s still incomplete?

Reply:
I think we can keep in the CONTRIBUTING and point people to it, perhaps if it's still hard to find we can move it to its own file :smile:.

I've also added it in aragon/aragon: https://github.com/aragon/aragon/pull/880.